fuzz.go

```diff
@@ -0,0 +1,30 @@
+// +build gofuzz
```

Comment:
If you're checking in the fuzzer, should you check in the corpus as well? In the past I've done this, under a `fuzz-data` subdirectory, so that it's easy to rerun the fuzzer without having to re-collect a bunch of samples.

With that said, I don't have a strong opinion, up to your preference.

Reply:
I debate a lot about this, but thus far, haven't checked it in anywhere.